runtime/parachains/src/initializer.rs

```diff
@@ -64,6 +74,14 @@ decl_storage! {
 		/// them writes to the trie and one does not. This confusion makes `Option<()>` more suitable for
 		/// the semantics of this variable.
 		HasInitialized: Option<()>;
+		/// Buffered session changes along with the block number at which they should be applied.
+		///
+		/// Typically this will be empty or one element long, with the single element having a block
+		/// number of the next block.
+		///
+		/// However this is a `Vec` regardless to handle various edge cases that may occur at runtime
+		/// upgrade boundaries or if governance intervenes.
```

Comment:
I guess this is what is needed in `Storage` :)

Reply:
I'm not sure it's needed in storage. we are basically saying the same thing in 2 places already. Now if one changes they will be inconsistent.